src/index.ts

```diff
@@ -9,17 +9,16 @@ function cx(...args: Argument[]): string;
 function cx(): string {
   let str = "",
     i = 0,
-    arg: unknown,
-    val: string | number;
+    arg: unknown;
 
   while (i < arguments.length) {
-    if ((arg = arguments[i++])) {
-      if (
-        (val = typeof arg === "string" || typeof arg === "number" ? arg : "")
-      ) {
-        str && (str += " ");
-        str += val;
-      }
+    if (
+      (arg = arguments[i++]) &&
+      arg &&
```

Comment:
Checking for falsy is a slight change of behavior, however `0` is not a valid class name so it probably shouldn't be present in the output.

Reply:
There is no change in behavior actually since the `(arg = arguments[i++])` is already itself a check for falsy! You could even omit the `arg &&`  to make the package smaller/faster. What do you think?

Reply:
Good point, I missed that! It's not any faster but it does save a few bytes 👌